Tests/AutoCollection/Statsbeat.tests.ts

```diff
@@ -171,17 +171,17 @@ describe("AutoCollection/Statsbeat", () => {
         it("Track counts", (done) => {
             statsBeat.enable(true);
             const sendStub = sandbox.stub(statsBeat, "_sendStatsbeats");
-            statsBeat.countRequest(0, "test", 1, true);
-            statsBeat.countRequest(0, "test", 1, true);
-            statsBeat.countRequest(0, "test", 1, true);
-            statsBeat.countRequest(0, "test", 1, true);
-            statsBeat.countRequest(0, "test", 1, false);
-            statsBeat.countRequest(0, "test", 1, false);
-            statsBeat.countRequest(0, "test", 1, false);
-            statsBeat.countRetry(0, "test");
-            statsBeat.countRetry(0, "test");
-            statsBeat.countThrottle(0, "test");
-            statsBeat.countException(0, "test");
+            statsBeat.countRequest(0, "test", 1, true, 200);
```

Comment:
StatusCode should be optional parameter if is not used all the time, like in this case

Reply:
If we make this parameter optional are we ok with creating entries in the request counter like `{ statusCode: undefined, count: <currentCount>`?

Reply:
no need to do that, successful count is only a number